nixos/modules/services/matrix/mautrix-discord.nix

```diff
@@ -0,0 +1,157 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
you should avoid this kind of use of with to address https://github.com/NixOS/nixpkgs/issues/208242 for this module.

use inherits where you find yourself excessively reusing `lib.foo`